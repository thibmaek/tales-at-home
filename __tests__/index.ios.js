import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import TalesAtHome from '../index.ios.js';

describe(`Container: <TalesAtHome />`, () => {
  it(`should render the container`, () => {
    const tree = renderer.create(<TalesAtHome />).toJSON();
    expect(tree).toMatchSnapshot(`TalesAtHome`);
  });
});
