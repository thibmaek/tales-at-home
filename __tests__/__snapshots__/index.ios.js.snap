exports[`TalesAtHome 1`] = `
<BVLinearGradient
  colors={
    Array [
      4282361830,
      4283223281,
    ]
  }
  end={undefined}
  locations={null}
  start={undefined}
  style={
    Object {
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    style={undefined}>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "white",
          "fontSize": 24,
          "textAlign": "center",
        }
      }>
      Kies een taal
    </Text>
    <View
      style={undefined}>
      <RCTPicker
        items={
          Array [
            Object {
              "label": "English",
              "textColor": undefined,
              "value": "en",
            },
            Object {
              "label": "Nederlands",
              "textColor": undefined,
              "value": "nl",
            },
          ]
        }
        onChange={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        selectedIndex={0}
        style={
          Array [
            Object {
              "height": 216,
            },
            undefined,
          ]
        } />
    </View>
  </View>
</BVLinearGradient>
`;
